### DISCUSSION TITLE
Convert "Answer" to "Comment"

### DISCUSSION THREAD

**Discussion by tim-rohrer** [community/community#5230] [Discussions]
Users often enter what should be a comment as a proposed answer in the Q&A category. This is really confusing, and doesn't promote good flow of discussion.

Is there a way I'm missing to convert a proposed answer to a comment of another answer? If not, can we get this capability added?

Thanks!

**Comment by hughlilly** [comment]
I agree; see my post #6179.

**Reply by hughlilly** [reply]
I suppose the problem is they're trying to imitate Stack Overflow, right? So until such time as an Answer is selected, every comment is a "suggested answer"— so I can see where the language comes from given that logic. I just don't think it's helpful to users, as you've rightly identified.

**Reply by VasanthGN** [reply]
@ddaaxz

**Comment by tonygermano** [comment]
Related to https://github.com/github/feedback/discussions/3798

**Comment by markseuffert** [comment]
> Users often enter what should be a comment as a proposed answer in the Q&A category. This is really confusing, and doesn't promote good flow of discussion.

We see the same. It's a recurring thing that new users click **Answer** instead of **Reply** for a follow-up question. Then users continue replying in the new thread, until there's an answer at the bottom of the new thread... which throws off the whole question/answer concept. Maybe it's a terminology problem? I discussed the behaviour with another maintainer and he mentioned that in German "answer" and "reply" are the same word. Actually in many European languages, I tried a couple in Google translate. 

It would be nice if moderators/maintainers could merge two threads together. See also #52.
